app/views/layouts/to_change_everything.html.erb

```diff
@@ -0,0 +1,261 @@
+<!DOCTYPE html>
+<html dir="<%=t('.language_direction')%>" lang="<%=t('.lang_code')%>" prefix="og: http://ogp.me/ns#" <%= html_id %>>
```

Comment:
I think I cargo culted `prefix="og: http://ogp.me/ns#" <%= html_id %>` from somewhere

I don't know what it is or if I need it

Reply:
open graph is meta tags use by Facebook, Tumblr and others websites to get information about an app or a page you share.

Reply:
@just1602 ah, ok 👍